tmt/steps/prepare/feature/__init__.py

```diff
@@ -186,6 +186,12 @@ class PrepareFeature(tmt.steps.prepare.PreparePlugin[PrepareFeatureData]):
         prepare:
             how: feature
             epel: disabled
+
+    .. note::
+
+       Features available via this plugin are implemented and shipped as
+       plugins too. See :ref:`/plugins/prepare-feature` for the list of
+       available feature plugins and their configuration.
```

Comment:
What about adding a similar mention to the /spec/plans/prepare section as well?
* https://tmt--3438.org.readthedocs.build/en/3438/spec/plans.html#feature